src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php

```diff
@@ -283,6 +283,16 @@ class SymfonyExcludeListPhpunit {}
     chdir($oldPwd);
 }
 
+// Create a symlink with a predictable path pointing to the currently used version.
+// This is useful for static analytics tools such as PHPStan having to load PHPUnit's classes
+// and for other testing libraries such as Behat using PHPUnit's assertions.
+chdir($PHPUNIT_DIR);
+if (file_exists('phpunit')) {
+    unlink('phpunit');
+}
+symlink($PHPUNIT_VERSION_DIR, 'phpunit');
```

Comment:
just in case the FS is read-only, this should be silenced I think (same for unlink)